ui/components/ui/text-field/text-field.component.js

```diff
@@ -19,6 +19,13 @@ const inputLabelBase = {
   color: 'var(--color-text-default)',
 };
 
+const fontFamily = [
+  '"Euclid Circular B"',
+  'Helvetica',
+  'Arial',
+  'sans-serif',
+].join(', ');
+
```

Comment:
i hate this. is there a better way to get our design-system's font family?

Reply:
Unfortunately, not really. We can use the typography object from the design tokens package, but it's still pretty janky. 

```javascript
import { typography } from '@metamask/design-tokens';

const fontFamily = [
  typography.fontFamilies.euclidCircularB,
  'Helvetica',
  'Arial',
  'sans-serif',
].join(', ');
```